src/metabase/driver.clj

```diff
@@ -963,6 +963,16 @@
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
 
+(defmulti column-name-length-limit
+  "Return the maximum number of characters allowed in a column name, or `nil` if there is no limit."
+  {:changelog-test/ignore true, :added "0.49.0", :arglists '([driver])}
```

Comment:
I plan to backport this twice. Even though 48 is still affected, I suspect it will have no further releases.

Reply:
D'oh, thanks for catching.